smoketest/harness/harness.go

```diff
@@ -236,6 +236,8 @@ func (h *Harness) Start() {
 	h.t.Helper()
 
 	var cfg config.Config
+	cfg.General.ApplicationName = "GoAlert"
```

Comment:
Instead of setting this, we should add a `Config.ApplicationName()` method that handles the default value